src/components/PortfolioAccordions.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div id="portfolio-accordions">
+    <h3 class="site-preview-heading">
+      Visualizations
+    </h3>
+
+    <div
+      class="usa-accordion"
+      aria-multiselectable="true"
+    >
+      <!-- Use the accurate heading level to maintain the document outline -->
+      <h4 class="usa-accordion__heading">
+        <button
+          class="usa-accordion__button"
+          aria-expanded="true"
+          aria-controls="m-a1"
+        >
+          Integrated Water Science Basins
+        </button>
+      </h4>
+      <div
+        id="m-a1"
+        class="usa-accordion__content usa-prose"
+      >
+        <div class="card-container grid-container">
+          <ul 
+            class="usa-card-group grid-row"
+          >
+            <PortfolioCard
+        
+              v-for="viz in vizList1"
+              :key="viz.title"
+              :viz="viz"
+              class="tablet:grid-col-4 usa-card"
+            />
+          </ul>
+        </div>
+      </div>
+
+      <!-- Use the accurate heading level to maintain the document outline -->
+      <h4 class="usa-accordion__heading">
+        <button
+          class="usa-accordion__button"
+          aria-expanded="false"
+          aria-controls="m-a2"
+        >
+          River Conditions
+        </button>
+      </h4>
+      <div
+        id="m-a2"
+        class="usa-accordion__content"
+      >
+        <div class="card-container grid-container">
+          <ul 
+            class="usa-card-group grid-row"
+          >
+            <PortfolioCard
+        
+              v-for="viz in vizList2"
+              :key="viz.title"
+              :viz="viz"
+              class="tablet:grid-col-4 usa-card"
+            />
+          </ul>
+        </div>
+      </div>
```

Comment:
For River Conditions, having the accordion show all iterations is a lot and the hurricane section gets sort of lost at the end. This list is going to continue to grow. I think this is where a carousel could be useful, or filtering to just the most recent few in the accordion.

Reply:
Another way to approach this could be to just have a single card that allows you to select the version you're looking for and by default goes to the most recent.

Reply:
I documented these thoughts in an issue https://github.com/USGS-VIZLAB/vizlab-home/issues/5